modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/progress.js

```diff
@@ -81,7 +81,11 @@ AUI.add(
 					var instance = this;
 
 					setTimeout(() => {
-						instance._frame.get('contentWindow.location').reload();
+						if (instance._frame.get('contentWindow')) {
```

Comment:
Could we use just `instance._frame.get('contentWindow.location')?.reload();`?

Reply:
Nop, the method `instance._frame.get` fails inside. It's doesn't expect `null` only `undefined`.
Maybe we can use something like `instance._frame._node.contentWindow?. location.reload()` but doesn't look good.

Reply:
Do you prefer to check the node directly `instance._frame._node.contentWindow?.location.reload()`? 
It's a little uglier for me.

Do have you another idea or suggestion?

Reply:
If we can do `instance._frame.get('contentWindow')?.location.reload()` I'd go for it, if not just keep your changes ;)

Reply:
I'm going to try it right now :D

Reply:
With a `?`  more works like a charm. Thanks!

`instance._frame.get('contentWindow')?.location?.reload()`